### ISSUE TITLE
[Bug]: New release of "kured" throws error when deploying

### ISSUE THREAD

**Issue by majkshkurti** [kube-hetzner/terraform-hcloud-kube-hetzner#919]
### Description

By default the kube.tf fetches the latest kured version. The version 1.32 (latest release) throws the following error: 

`error: accumulating resources: accumulation err='accumulating resources from 'https://github.com/weaveworks/kured/releases/download/1.13.2/kured-1.13.2-dockerhub.yaml': may not add resource with an already registered id: ServiceAccount.v1.[noGrp]/kured.kube-system': evalsymlink failure on '/tmp/kustomize-2026577410/releases/download/1.13.2/kured-1.13.2-dockerhub.yaml' : lstat /tmp/kustomize-2026577410/releases: no such file or directory`

### Kube.tf file

```terraform
module "kube-hetzner" {
  providers = {
    hcloud = hcloud
  }
  hcloud_token    = var.hcloud_token
  source          = "kube-hetzner/kube-hetzner/hcloud"
  ssh_public_key  = file(var.kube_hetzner_ssh_public_key)
  ssh_private_key = file(var.kube_hetzner_ssh_private_key)
  network_region  = "eu-central"
  cluster_name    = "dev"
  control_plane_nodepools = [
    {
      name        = "control-plane-nbg1",
      server_type = "cpx11",
      location    = "nbg1",
      labels      = [],
      taints      = [],
      count       = 1
    },
  ]
  agent_nodepools = [
    {
      name        = "agent-large",
      server_type = "cpx31",
      location    = "nbg1",
      labels      = [],
      taints      = [],
      count       = 1
    }
  ]
  load_balancer_type       = "lb11"
  load_balancer_location   = "nbg1"
  ssh_max_auth_tries       = 15
  automatically_upgrade_os = false
  create_kubeconfig        = false
  # kured_version            = "1.13.1"
}
```

### Platform

Linux

**Comment by Gh0stRocket** [comment]
I have the same problem and am glad to help troubleshoot.

**Comment by majkshkurti** [comment]
It seems that the ServiceAccount is defined twice in the kured manifest: 

![image](https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/assets/32641116/7eeabd3b-cc09-44e1-8ea5-8f0b37485436)

**Comment by mysticaltech** [comment]
> And there are also other resources which are removed from yaml in this release.
> We can explicitly define `kured_version = "1.13.1"` at the moment

Exactly. @majkshkurti offers the solution above to this temporary problem coming from kured, just set the version with `kured_version`.

**Comment by mysticaltech** [comment]
I temporarily fixed the default version of kured to 1.13.1, this fix shipped in the latest release v2.5.0.
